client/src/app/auth/services/auth.service.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { tap } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  constructor(private http: HttpClient) {}
+
+  login(email: string, password: string) {
+    return this.http
+      .post('http://localhost:3000/users/login', {
+        email,
+        password,
+      })
+      .pipe(tap((res) => this.setSesstion(res)));
+  }
+
+  isLogged(): boolean {
```

Comment:
Name `isLogged` suits for variable, but is misleading name for method. Please look for more suitable name, for example `getIsLoggedStatus` or similar